tools/distpartitioning/dataset_utils.py

```diff
@@ -164,6 +164,54 @@ def get_dataset(input_dir, graph_name, rank, world_size, schema_map):
         for k, v in node_features.items():
             logging.info(f'[Rank: {rank}] node feature name: {k}, feature data shape: {v.size()}')
 
+    '''
+    Reading edge features now.
+    '''
+    edge_features = {}
+    edge_feature_tids = {}
+
+    #iterate over the "edge_data" dictionary in the schema_map
+    #read the edge features if exists
+    #also keep track of the type_eids for which the edge_features are read.
+    dataset_features = schema_map[constants.STR_EDGE_DATA]
+    if((dataset_features is not None) and (len(dataset_features) > 0)):
+        for etype_name, etype_feature_data in dataset_features.items():
+            #etype_feature_data is a dictionary
+            #where key: feature_name, value: dictionary in which keys are "format", "data"
+            edge_feature_tids[etype_name] = []
+            for feat_name, feat_data in etype_feature_data.items():
+                assert len(feat_data[constants.STR_DATA]) == world_size
+                assert feat_data[constants.STR_FORMAT][constants.STR_NAME] == constants.STR_NUMPY
+                my_feat_data_fname = feat_data[constants.STR_DATA][rank] #this will be just the file name
```

Comment:
my_feat_data_fname?
Let's avoid word like 'my' / 'your' e.t.c in the code and comment? All the code / comment should be objective, regardless who are the people to read and write them.

Reply:
This is a typical way of prefixing variable names to indicate that this stores information which is processed by current process in the distributed environment.

Reply:
then use "local_xxx" or "cur_xxx".

Reply:
Changed the variable name as suggested. Now it is treated as ordinary file name as suggested by Rhett-Ying.